shared/reducers/config.js

```diff
@@ -190,6 +192,12 @@ export default function (state: ConfigState = initialState, action: Action): Con
       }
     }
 
+    case 'app:changedFocus':
+      return {
+        ...state,
+        appFocused: action.payload.focused,
```

Comment:
maybe make the action.payload.appFocused to be consistent with same name everyhwere